app/views/layouts/_manage_nav.html.erb

```diff
@@ -8,10 +8,11 @@
       <%= link_to 'Manage Variables', manage_variables_path %>
     </h2>
 
-    <!-- Manage Services Link Commented out until functional implementation ready. This will also need style treatment as all links do not fit in header -->
+    <% if Rails.configuration.umm_s_enabled %>
     <h2 class="<%= 'current' if current_manage_title == 'manage_services' %>">
```

Comment:
not a huge deal, but I thought we were indenting html content within erb `if` statements